Content review and update (#26)
ministryofjustice/mkdocs-tech-docs-template: docs/setup/README.md

# Quickstart guide

This page provides a guide on how to configure the MKDocs Tech Docs Template. In a few simple steps, you can begin building your documentation site.

## Before you begin

In order to use the MKDocs Tech Docs Template, you require the following:

- a device with permissions to install new packages
- Python 3 and Pip installed on your device
- GitHub account with [a configured SSH key][gh_ssh]
- GitHub repository to store your documentation (see [GitHub's documentation][gh_newRepo] for instructions)

> 

Reviewer: You don't need a GitHub account or a GitHub repository, for example you could use BitBucket

You should also be comfortable writing content in Markdown, and using the Git workflow to push your changes to a repository.

## Install MKDocs TDT

> The MkDocs TDT publishes as a Python package, so you can use pip to install it. In your terminal, navigate to the directory you want to install the template and run the following:

Reviewer: ```suggestion
The MkDocs TDT publishes as a Python package. In your terminal, navigate to the directory you want to install the template and run the following to install using pip:
```
